build/includes/terraform.mk

```diff
@@ -15,17 +15,19 @@
 # The GKE development cluster name
 GCP_TF_CLUSTER_NAME ?= agones-tf-cluster
 
-# the current project
-current_project := $(shell $(DOCKER_RUN) bash -c "gcloud config get-value project 2> /dev/null")
```

Comment:
Just realised we're deleting `current_project` - this will break stuff :cry:

Reply:
Oops, sorry for that, returning it back.